### DISCUSSION TITLE
在一个全新的vue&vant项目中，导入官方模板，无法渲染，无法正常使用

### DISCUSSION THREAD

**Discussion by yuyuabc1** [alibaba/formily#3662] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/26144081/210479683-82c4e1b2-caf2-4584-87ff-431cfcad1ae6.png)

使用的package.json版本：
    "@formily/core": "^2.2.14",
    "@formily/element": "^2.2.14",
    "@formily/vant": "^1.0.0-beta.14",
    "@formily/vue": "^2.2.14",
    "@vue/composition-api": "^1.7.1",
    "ant-design-vue": "^3.2.15",
    "core-js": "^3.8.3",
    "element-ui": "^2.15.12",
    "less": "^4.1.3",
    "less-loader": "^11.1.0",
    "sass": "^1.32.11",
    "sass-loader": "^8.0.2",
    "vant": "^2.12.53",
    "vue": "^2.6.14",
    "vue-router": "^3.6.5"

node版本：
v14.15.0

感觉对vue生态非常不友好

**Comment by janryWang** [comment]
导入什么官方模板？

**Reply by yuyuabc1** [reply]
https://vant.formilyjs.org/

下载依赖，然后再一个页面中copy上述代码

**Reply by yuyuabc1** [reply]
现已查明 @formily/vant无法在vue@2.7.x版本中运行，需将项目降级至vue@2.6.x方可运行
